guides/source/form_helpers.md

```diff
@@ -104,8 +104,8 @@ ending in `_tag` (such as `text_field_tag` and `check_box_tag`), generate just a
 single `<input>` element. The first parameter to these is always the name of the
 input. When the form is submitted, the name will be passed along with the form
 data, and will make its way to the `params` hash in the controller with the
```

Comment:
There are a lot of other references to `params` as a hash in the guides. Would you like me to drop the word hash in all of them or change hash to object?